docs/dataset_licensing.md

```diff
@@ -0,0 +1,118 @@
+| Modification | (Derivative work) Creation of adversarial dataset that modifies the original images with small perturbations that are crafted to fool machine learning models.  |
+
+### Imagenette
+|Attribution                   |              |  
+|------------------------------|--------------|
+| Creator/attribution parties  | Jeremy Howard  |
+| Copyright notice             | Copyright &copy; 2019 by Jeremy Howard |
+| Public license notice        | https://github.com/fastai/imagenette |
+| Disclaimer notice            | Disclaimer of Warranty. Unless required by applicable law or agreed to in writing, Licensor provides the Work (and each Contributor provides its Contributions) on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE. You are solely responsible for determining the appropriateness of using or redistributing the Work and assume any risks associated with Your exercise of permissions under this License. |
+| Dataset link | https://github.com/fastai/imagenette |
+| Modification | (Slight) Representation of images as binary tensors |
+
+## Fair use notes for RESISC-45 and UCF101
+* Two Six Labs does not charge users for access to the Armory repository, 
+nor the datasets therein, nor does it derive a profit directly from use of the 
+data sets.
+* Two Six Labs is not merely republishing the original datasets. The 
+datasets have undergone transformative changes, specifically they have been 
+repackaged to be integrated with Tensorflow Datasets. This repackaging 
+includes, but is not limited to, processing images from compressed formats into 
+binary tensors as well as decoding audio and video files. Further, Two Six Labs 
+has published derived adversarial datasets that modify the original images/videos with 
+small perturbations that are crafted to fool machine learning models for both 
+the RESISC-45 and UCF101 datasets.
+* Two Six Labs uses these data sets within Armory, however there are 
+other additional datasets present, as well as multiple other features present 
+in Armory beyond providing datasets.
+* Two Size Labs provides public benefit through the public distribution 
+of the Armory framework to evaluate machine learning models. This material is 
+based upon work supported by the Defense Advanced Research Projects Agency 
+(DARPA) under Contract No. HR001120C0114. Any opinions, findings and 
+conclusions or recommendations expressed in this material are those of the 
+author(s) and do not necessarily reflect the views of the Defense Advanced 
+Research Projects Agency (DARPA).
+
```

Comment:
I might also add two bullet points, something to the effect to:
* Two Six Labs attempted to contact the authors of RESISC-45, but received no response.
* UCF101 direct download functionality has been used by other ML frameworks, such as TensorFlow: https://www.tensorflow.org/datasets/catalog/ucf101